azure-pipelines.yml

```diff
@@ -41,11 +41,6 @@ steps:
     echo "##vso[task.setvariable variable=PBHOSTURL]$(echo "${PBURL##*://}")"  
   displayName: Set up Host URL
 
-# - bash: |
-#     echo "51.11.46.27    $PBHOSTURL" | sudo tee -a /etc/hosts
-#     curl -k -i $(PBURL) --fail --connect-timeout 30
-#   displayName: 'Check URL is reachable'
```

Comment:
Will you be reinstating this at some point?

Reply:
nope, I'll remove

Reply:
This check is now done at the Selenium Grid setup step (checks the nodes can connect to the environment)